users/models.py

```diff
@@ -32,12 +33,32 @@ class Meta:
         verbose_name_plural = "Allergies"
 
 
+class KnightHood(models.Model):
+    """
+    Model for knighthood. A knighthood is a special title given to a user
+    """
+
+    user = models.OneToOneField(
+        "users.User", on_delete=models.CASCADE, related_name="knighthood"
+    )
+    knighted_at = models.DateField(default=date.today())
+    description = models.TextField(blank=True)
+
+    def __str__(self):
+        return f"Knighthood for {self.user}"
+
+    def __repr__(self):
+        return f"Knighthood(user={self.user})"
+
+
 class User(AbstractUser):
-    nickname = models.CharField(max_length=64, blank=True, null=True, default=None)
+    nickname = models.CharField(
```

Comment:
Er dette black formattering? If not gjerne kjør en format runde med det så får man ikke masse sære formating differ